src/routes/resources.js

```diff
@@ -14,10 +14,10 @@ module.exports = function Resources(app, model) {
   const modelName = Implementation.getModelName(model);
 
   this.list = (request, response, next) => {
-    const params = request.query;
-    const fieldsPerModel = new ParamsFieldsDeserializer(params.fields).perform();
+    const { query, user } = request;
+    const fieldsPerModel = new ParamsFieldsDeserializer(query.fields).perform();
 
-    return new Implementation.ResourcesGetter(model, lianaOptions, params)
+    return new Implementation.ResourcesGetter(model, lianaOptions, query, user)
```

Comment:
[MISC] Ins't it inconsistent to have `query` here and `params` for the `ResourceGetter (I might be missing something though)